docs/content/create-medusa-app.mdx

```diff
@@ -13,17 +13,15 @@ In this document, you’ll learn how to use `create-medusa-app` to create a Medu
 
 ## Overview
 
-Medusa is composed of three different components: the headless backend, the storefront, and the admin dashboard.
+Medusa is composed of three different components: a Node.js backend, the storefront, and the admin dashboard.
```

Comment:
suggestion: We should rewrite this overview section to align with our updated positioning. What I mean is Medusa is no longer only composed of three different components, but we are now moving toward being a toolkit for any type of commerce application. 

I can write out a suggestion, if you think it makes sense.

Reply:
Thanks for the suggestion! I think that makes a lot of sense. I changed up the overview section to reflect that, but feel free to make suggestions for additional changes in it